CHANGELOG.md

```diff
@@ -1,58 +1,63 @@
-
-- Install system dependencies on Linux.
-
-- Use Use failure() to trigger steps on failures
-
-- `SymPair::sym_pair()` now returns `(Option<Robj>, Robj)`
-
-- Bump `ndarray` to 0.15.3. Under [RFC 1977](https://github.com/rust-lang/rfcs/pull/1977) this is a "public dependency" change, and therefore can be considered a breaking change, as consumers of `extendr` that use an older version of `ndarray` will no longer be compatible until they also bump `ndarray` to a compatible version.
-
-- Implement `TryFrom<&ArrayBase> for Robj`, allowing `extendr`-annotated functions to return Arrays from the `ndarray` crate and have them automatically converted to R arrays. Note, even if 1D arrays are returned they will not be returned as vectors.
-
-- Removed `TryFrom` conversions between `Robj` and `HashMap` for consistency. `List::into_hashmap()` and `List::from_hashmap()` should be used instead.
-
-- Added support for the graphics-related API (`graphics` feature flag, disabled by default).
+## 0.3.0
+
+### Added
+
+- `Function` type that wraps an R function, which can be invoked using the `call()` method [[#188]](https://github.com/extendr/extendr/pull/188)
+- `pairlist!` macro for generating `Pairlist` objects, e.g. for use in function calls [[#202]](https://github.com/extendr/extendr/pull/202)
+- `use_try_from` option for the `extendr` macro, which allows the use of any type that implements `TryInto<Robj>`/`TryFrom<Robj>`, e.g. `#[extendr(use_try_from = true)]` [[#222]](https://github.com/extendr/extendr/pull/222)
+- Large Rust integer types (`u32`, `u64` and `i64`) can now be converted to R's `numeric` type, which can handle large integer values [[#242]](https://github.com/extendr/extendr/pull/242)
+- `call!` macro, which can be used to call an R function whose name is provided as a string [[#238]](https://github.com/extendr/extendr/pull/238)
+- Implemented `TryFrom<Robj>` for a large number of Rust types [[#249]](https://github.com/extendr/extendr/pull/249), [[#258]](https://github.com/extendr/extendr/pull/258)
+- Support for `ALTREP` [[#250]](https://github.com/extendr/extendr/pull/250)
+- An `S4` struct, which wraps an S4 class in R [[#268]](https://github.com/extendr/extendr/pull/268)
+- Implemented `TryFrom<&ArrayBase> for Robj`, allowing `extendr`-annotated functions to return Arrays from the `ndarray` crate and have them automatically converted to R arrays. Note, even if 1D arrays are returned they will not be returned as vectors. [[#275]](https://github.com/extendr/extendr/pull/275)
+- `Rint`, `Rdouble`, `Rbool` and `Rcplx`: `ALTREP` wrapper classes for R vectors, which can be used without copying [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
```

Comment:
Those aren't `ALTREP`, thay are `NA`-aware (zero-cost) wrappers around primitive scalars like `i32`, `f64`, etc.

Reply:
So they're wrappers around Rust numeric types that can be used in R? Do they get converted to native vectors?